joomla/joomla-cms
# Improve accessibility for ISIS

### Summary of Changes

Add a skip to main content link to the isis template. As this is useful for keyboard users AND users with screenreaders the link is made visible when it has focus
### Testing Instructions

Apply the PR and go to ANY page and press tab and the link will appear. Hit the enter key and you will jump to the beginning of the toolbar

On the admin home page you jump to the first item on the left hand quicklinks

![a11yb](https://cloud.githubusercontent.com/assets/1296369/17836827/2fb2c15c-6797-11e6-900a-bcd64fe80c5d.gif)

![a11ya](https://cloud.githubusercontent.com/assets/1296369/17836828/2fb73ade-6797-11e6-8071-99c6e527b9b0.gif)
### Documentation Changes Required

None
